_translator-files/po/de/general.po

```diff
@@ -54,14 +56,13 @@ msgstr "Kommentare"
 #: ../wiki/en/misc/general.yml:0
 #, no-wrap
 msgid "<strong>Community knowledge base</strong><br /> Feel free to add your own article!"
-msgstr "<strong>Community Knowledge Base</strong><br /> Füge deinen eigenen Artikel hinzu!"
+msgstr "<strong>Jamulus Wissensdatenbank</strong><br /> Füge deinen eigenen Artikel hinzu!"
 
 #. type: Hash Value: kb mainpage
 #: ../wiki/en/misc/general.yml:0
-#, fuzzy, no-wrap
-#| msgid "KB overview"
+#, no-wrap
 msgid "Knowledge Base overview"
-msgstr "KB Übersicht"
+msgstr "Überblick über die Wissensdatenbank"
```

Comment:
I'd leave it as Übersicht or KB Übersicht. It's a headline which looks strange with too much text

Reply:
Corrected to: "Knowledge Base Übersicht"
OK?

Reply:
Yes.